examples/toy-model-stochastic-degradation.ipynb

```diff
@@ -0,0 +1,190 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Stochastic degradation model\n",
+    "\n",
+    "This example shows how the stochastic degradation model can be used.\n",
+    "This model describes the stochastic process of a single chemical reaction, in which the concentration of a substance degrades over time as particles react.\n",
+    "The substance degrades starting from an initial concentration, n_0, to 0 following a rate constant, k, according to the following model (Erban et al., 2007):\n",
```

Comment:
Can you make sure all the notation are consistent? E.g. n_0 here, you can do $n_0$ to make it equation-like, and so as k to $k$ etc.